akka-http-testkit/src/main/scala/akka/http/javadsl/testkit/RouteTest.scala

```diff
@@ -76,5 +77,7 @@ abstract class RouteTest extends AllDirectives with WSTestRequestBuilding {
       def run(request: HttpRequest): TestRouteResult = runRoute(underlying, request)
     }
 
-  protected def createTestRouteResult(request: HttpRequest, result: RouteResult): TestRouteResult
+  protected def createTestRouteResult(request: HttpRequest, result: RouteResult): TestRouteResult =
```

Comment:
Keeping `createTestRouteResult` allows to stay more compatible, but this is a low-level API so might not make sense.